charts/add-ons/grafana/templates/grafana.yaml

```diff
@@ -147,7 +147,15 @@ spec:
         - mountPath: /etc/grafana
           name: grafana-config
           readOnly: true
+      {{- if not (empty .Values.proxy.resources) }}
```

Comment:
Since `.Values.proxy` is not necessarily defined (which is the default scenario), you'll need to check first that it's not empty, or else checking for `.Values.proxy.resources` gives a nil pointer error